Contents/Code/plex_api_helper.py

```diff
@@ -404,9 +405,9 @@ def get_database_info(item):
                     database_id = temp_database_id
                     database = temp_database
 
-                if temp_database == 'themoviedb':
+                if temp_database in ('themoviedb', 'tmdb'):
```

Comment:
```suggestion
                if temp_database in ('themoviedb', 'tmdb'):  # tmdb is our prefered db, so we break if found
```

Also... is the new agent returning 2 different things for the movie db?

Reply:
Seems so

![image](https://github.com/LizardByte/Themerr-plex/assets/4533568/f42e2b02-43c8-4eb1-8719-289511255927)